api.go

```diff
@@ -887,6 +986,15 @@ func (lapi *Client) GetAsset(id string) (*Asset, error) {
 	return &asset, nil
 }
 
+func (lapi *Client) ListAssets() (*[]Asset, error) {
+	var assets []Asset
+	url := fmt.Sprintf("%s/api/asset/", lapi.chosenServer)
```

Comment:
Even though it's interpreted correctly by the server, the list API does not have a trailing slash (could be interpreted as a GET of an empty ID instead). Should be just
```suggestion
	url := fmt.Sprintf("%s/api/asset", lapi.chosenServer)
```